src/functions.sh

```diff
@@ -50,11 +50,11 @@ sqm_logger() {
             echo "$@" >&2
         fi
     fi
-    # slightly dangerous as this will keep adding to the log file
+    
+    # this writes into SQM_START_LOG or SQM_STOP_LOG, log files are trucated in 
+    # start-sqm/stop-sqm respectively and should only take little space
     if [ "$debug" -eq "1" ]; then
-        if [ "$level_max" -ge "$LEVEL" -o "$LEVEL" -eq "$VERBOSITY_TRACE" ]; then
-            echo "$@" >> ${SQM_DEBUG_LOG}
-        fi
```

Comment:
Why remove the if?

Reply:
I see no reason to make the content of the log dependent on level_max, we do this to avoid spamming the openwrt log, but here it seems easiest to simply log everything. Again this is from the perspective of being able to ask users to post the log files with minimal iterations.

Reply:
Yeah, fair enough...